src/harness/fake_sas.py

```diff
@@ -260,6 +260,9 @@ def TriggerDailyActivitiesImmediately(self):
   def GetDailyActivitiesStatus(self):
     return {'completed': True}
 
+  def InjectCpiUser(self, request):
```

Comment:
You can safely remove this function since it's never called. (We only define these functions if we need to return an "interesting" response.)